### PR TITLE
rename roamer structs

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: include/roamer.h [hunk 1]
```diff
@@ -25,38 +25,38 @@ typedef struct ROAMER {
...
-void RoamerSave_SetFlute(ROAMER_SAVE *roamerSave, u8 a1);
-u8 RoamerSave_GetFlute(ROAMER_SAVE *roamerSave);
+u8 RoamerSave_OutbreakActive(RoamerSaveData *roamerSave);
+void PlayerLocationHistoryPush(RoamerSaveData *roamerSave, u32 mapsec);
+u32 PlayerLocationHistoryGetBack(RoamerSaveData *roamerSave);
+u8 Roamer_GetLocation(RoamerSaveData *roamerSave, int a1);
+void Roamer_SetLocation(RoamerSaveData *roamerSave, int a1, u8 a2);
+u8 GetRoamerIsActiveByIndex(RoamerSaveData *roamerSave, int a1);
+void RoamerMon_Init(Roamer ** roamer_p);
+Roamer *Roamers_GetRoamMonStats(RoamerSaveData *roamerSave, int a1);
```
Comment (adrienntindall) [inline include/roamer.h:54]: Romaers->RoamerSave(Data)?

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: include/roamer.h (hunk 1 above)
Comment (red031000) [reply include/roamer.h:54]: you mean for the func names?

Comment (adrienntindall) [reply include/roamer.h:54]: yeah, since having Roamer_ and Roamers_ prefixes don't make that much sense

Comment (red031000) [reply include/roamer.h:54]: which ones specifically do you think should be changed?

Comment (adrienntindall) [reply include/roamer.h:54]: Roamers_ seems to refer to relating to the RoamerSaveData struct right? Where Roamer_ refers to the Roamer struct?

Comment (red031000) [reply include/roamer.h:54]: ok, let me rephrase, For Save_Roamers_Init, what do you think would be best to replace that with

**Review by adrienntindall — APPROVED** [review]
